firstvoices/backend/models/media.py

```diff
@@ -0,0 +1,158 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+from ..permissions import predicates
+from .base import AudienceMixin, BaseSiteContentModel
+
+
+class Person(BaseSiteContentModel):
+    # from FVContributor
+
+    class Meta:
+        verbose_name = _("Person")
+        verbose_name_plural = _("People")
+        rules_permissions = {
+            "view": predicates.has_visible_site,
+            "add": predicates.is_superadmin,  # permissions will change when we add a write API
+            "change": predicates.is_superadmin,
+            "delete": predicates.is_superadmin,
+        }
+
+    # from dc:title
+    name = models.CharField(max_length=200)
+
+    # from FVContributor dc:description
+    bio = models.CharField(max_length=500)
+
+    def __unicode__(self):
+        return f"{self.name} ({self.site})"
+
+
+def media_directory_path(instance, filename):
+    # file will be uploaded to MEDIA_ROOT/<site slug>/<filename>
+    return f"{instance.site.slug}/{filename}"
+
+
+class MediaBase(AudienceMixin, BaseSiteContentModel):
+    class Meta:
+        abstract = True
+
+    # see specific media models for migration info
+    acknowledgement = models.TextField(max_length=500, blank=True)
+
+    # from dc:title
+    title = models.CharField(max_length=200)
+
+    # from dc:description
+    description = models.CharField(max_length=500, blank=True)
+
+    # exclude_from_games from fv-word:available_in_games, fvaudience:games
+
+    # exclude_from_kids from fvaudience:children fvm:child_focused
+
+    # from fvm:shared
+    is_shared = models.BooleanField(default=False)
+
+    # from fvm:content
+    content = models.FileField(upload_to=media_directory_path)
```

Comment:
Note: attributes like size, filename, mime type, and image dimensions can be retrieved from the FileField itself, so there are not separate fields for storing them